NuGet.config

```diff
@@ -4,7 +4,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
+    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
I picked the url from roslyn master NuGet.config: https://github.com/dotnet/roslyn/blob/master/NuGet.Config#L24